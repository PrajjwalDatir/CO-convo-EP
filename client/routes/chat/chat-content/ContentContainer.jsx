import React from 'react';
import PropTypes from 'prop-types';

function ContentContainer({ children, direction }) {
  return (
    <div
      className={`my-8 flex flex-grow bg-white rounded-lg shadow-xl ${direction}`}
    >
      {children}
    </div>
  );
}

ContentContainer.propTypes = {
  children: PropTypes.node,
  direction: PropTypes.string,
};

ContentContainer.defaultProps = {
  children: null,
  direction: 'flex-row',
};

export default ContentContainer;
