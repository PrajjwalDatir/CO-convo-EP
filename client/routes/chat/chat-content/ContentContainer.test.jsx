import React from 'react';
import { mount } from 'enzyme';
import ContentContainer from './ContentContainer';

describe('<ContentContainer />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<ContentContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
