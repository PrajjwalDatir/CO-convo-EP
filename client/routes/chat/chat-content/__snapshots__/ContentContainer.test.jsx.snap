// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentContainer /> should match snapshot 1`] = `
<ContentContainer
  direction="flex-row"
>
  <div
    className="my-8 flex flex-grow bg-white rounded-lg shadow-xl flex-row"
  />
</ContentContainer>
`;
